tensorflow/tensorflow.bzl

```diff
@@ -410,9 +417,9 @@ def tf_binary_dynamic_kernel_deps(kernels):
 # TODO(pcloudy): Remove this workaround when https://github.com/bazelbuild/bazel/issues/4570
 # is done and cc_shared_library is available.
 SHARED_LIBRARY_NAME_PATTERNS = [
-    "lib%s.so",  # On Linux, shared libraries are usually named as libfoo.so
-    "lib%s.dylib",  # On macos, shared libraries are usually named as libfoo.dylib
-    "%s.dll",  # On Windows, shared libraries are usually named as foo.dll
+    "lib%s.so%s",  # On Linux, shared libraries are usually named as libfoo.so
+    "lib%s%s.dylib",  # On macos, shared libraries are usually named as libfoo.dylib
+    "%s%s.dll",  # On Windows, shared libraries are usually named as foo.dll
```

Comment:
As far as I understand, there is no versioned DLL files on Windows, can you exclude Windows from those logic?

Reply:
done